src/ui/components/Rating/index.js

```diff
@@ -9,7 +9,7 @@ import translate from 'core/i18n/translate';
 import './styles.scss';
 
 
-const RATING_STYLES = ['small', 'large', 'small-monochrome'];
+const RATING_STYLES = ['small', 'large', 'small-monochrome', 'small-by-user'];
```

Comment:
As mentioned, I think we should decouple ownership from size. I guess we sort of messed up by having the "small monochrome" one already (are we even using it anywhere?)

But if we can have them be separate I think that'd be better.